samples/AzureBlob/Program.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using Microsoft.AspNetCore.DataProtection;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.WindowsAzure.Storage;
+
+namespace AzureBlob
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            var storageAccount = CloudStorageAccount.DevelopmentStorageAccount;
+            var client = storageAccount.CreateCloudBlobClient();
+            var container = client.GetContainerReference("key-container");
+
+            // The container must exist before calling the DataProtection APIs.
+            // The specific file within the container does not have to exist,
+            // as it will be created on-demand.
```

Comment:
This seems like a messy implementation to me. Why wasn't this setup to take a storage account name and key [or a connection string (e.g., `DefaultEndpointsProtocol=https;AccountName=NAME;AccountKey=KEY`)] and the container name and that's all that the dev needs to do? It would be great if you would get all of this into `.PersistKeysToAzureBlobStorage()`.